scripts/contributors/index.es

```diff
@@ -2,33 +2,59 @@ import fs from "fs"
 import path from "path"
 
 import color from "chalk"
-import async from "async"
-import PromisePolyfill from "promise"
-import lodash from "lodash"
+import {denodeify as asyncify} from "promise"
+
 import GithubApi from "github"
 
 import logger from "../utils/logger"
 
-const exec = PromisePolyfill.denodeify(require("child_process").exec)
-const glob = PromisePolyfill.denodeify(require("glob"))
-const readFile = PromisePolyfill.denodeify(fs.readFile)
-const writeFile = PromisePolyfill.denodeify(fs.writeFile)
+const exec = asyncify(require("child_process").exec)
+const glob = asyncify(require("glob"))
+const readFile = asyncify(fs.readFile)
+const writeFile = asyncify(fs.writeFile)
 
-const log = logger("contributors")
+// //
+
+const authorsFiles = "content/authors/*.json"
+const contributorsFile = "contributors.json"
 
-const githubApi = new GithubApi({version: "3.0.0"})
+const githubApi = new GithubApi({
+  version: "3.0.0",
+  headers: {
+    "user-agent": "putaindecode",
+  },
+})
+
+if (process.env.GITHUB_TOKEN || process.env.GH_TOKEN) {
```

Comment:
With this trick, I think we can drop the versionned contributors.json (even on Travis), but new players (or people that doesn't have a defined token) will be stuck by github api limitation when not authentified.
So I guess we should keep the cache for now.